ScrumptiousJellyfish/WorkforceManagement
# Added details functionality to EmployeeController

Please structure your pull requests this way:
- **Purpose of PR**
to add details functionality to EmployeeController
&nbsp;


- **Issue number this request impacts** ------
#5 

&nbsp;

- **Description of what was changed:**
added details method on EmployeeController and edited the Details view to display the information requested by the ticket

&nbsp;

- **Thorough list of steps to test**
1. pull down the branch ```git checkout jw-employee-details```
2. click the green triangle button in visual studio **OR** run ```dotnet run``` and navigate to ```http://localhost:xxxx/Employee``` in the browser
3. on the webpage click ```details``` on any of the items and it should load a details page with the following information: Employee Name, Department Name, Model Name, Program Name
&nbsp;


if you don't do this you will be
**FIRED**
